dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/LKSGData.kt

```diff
@@ -0,0 +1,221 @@
+package org.dataland.datalandbackend.model.lksg
```

Comment:
What is this file and why is it not displayed in the IDE?

Reply:
I deleted this file via GitHub commit.
I think it's not displayed in the IDE because it only exists on remote

Reply:
Ok